scss/custom/_card.scss

```diff
@@ -173,13 +173,13 @@ a.card-body { /* stylelint-disable-line selector-no-qualifying-type */
 
   .card {
     display: flex;
-    flex-basis: 17.5rem;
+    flex-basis: 15.3rem;
```

Comment:
why exactly 15.3 rem? would 15.5 / 15 not work out?

Reply:
That's because of the margins.

Reply:
> why exactly 15.3 rem? would 15.5 / 15 not work out?

That's because of the margins.